server/s3_motr_rw_common.cc

```diff
@@ -89,7 +98,32 @@ void motr_op_done_on_main_thread(evutil_socket_t, short events,
              S3_IEM_MOTR_CONN_FAIL_JSON);
     }
     context->on_failed_handler()();  // Invoke the handler.
+
+    if (error_code == -ETIMEDOUT) {
```

Comment:
Have a comment which talks about the logic

Reply:
done